src/app/(auth)/login/page.tsx

```diff
@@ -23,20 +23,24 @@ export default function Login() {
     });
 
     if (error) {
-      throw new Error(`An error occurred trying to sign in: ${error.message}`);
+      setErrorMessage(error.message);
+    } else {
+      push('/');
     }
-    push('/');
   };
 
   return (
     <>
-      <H1>Log In</H1>
-      <SpacerDiv>
-        <QuestionsDiv>
+      <SpacerDiv $gap={0.625}>
+        <H1>Log In</H1>
+        {errorMessage !== '' && <P $color={COLORS.redMid}>{errorMessage}</P>}
+      </SpacerDiv>
+      <SpacerDiv $gap={13 / 16}>
+        <SpacerDiv>
           <TextInput
             label="Email"
             placeholder="email@example.com"
-            errorText="" // "Email Error"
+            errorText={email === '' ? 'Invalid Email' : ''}
```

Comment:
(cc @davidqing6432) instead of immediately checking for potential empty email/password fields, can we only check for this when the "Log In" button is pressed? i think having the page load with error fields already in place might be a bit confusing.

Reply:
sounds good! yeah i agree that having them appear initially is confusing